controllers/article_id.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../db/connection')
+const { fetchArticleById } = require('../models/article_id')
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    fetchArticleById(article_id)
+        .then((article) => {
+            res.status(200).send({
+                author: article.author,
+                title: article.title,
```

Comment:
because you're not renaming any of the properties in article you can save a lot of lines of code here by just sending `({ article })`